test_image_analysis.py

```
@@ -0,0 +1,250 @@
+        "model": "gpt-4o",
+        "stream": True,
+        "images": [
+            "https://upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Gfp-wisconsin-madison-the-nature-boardwalk.jpg/2560px-Gfp-wisconsin-madison-the-nature-boardwalk.jpg"
+        ],
+        "conversation_history": [
+            {"role": "system", "content": "You are a helpful assistant that analyzes images."}
+        ]
+    }
+
+    try:
+        response = requests.post(url, json=payload, stream=True)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)